docs/content/docs/connectors/table/hive/hive_read_write.md

```diff
@@ -472,6 +472,14 @@ This configuration is set in the `TableConfig` and will affect all sinks of the
   </tbody>
 </table>
 
+### Configuration for Dynamic Partition Inserting 
+By default, if it's for dynamic partition inserting, Flink will sort the data additionally by dynamic partition columns before writing into sink table.
+
+To avoid the extra sorting, you can set job configuration `table.exec.hive.dynamic-grouping.enabled` (`true` by default) to `false`.
+But with such configuration, it'll throw OOM exception if there are too may dynamic partitions.
+
```

Comment:
Add some hints about how to tune the dynamic partitioning. For example, add `DISTRIBUTED BY <partition_fields>` for hash shuffling when data is not skewed. You can also manually add `SORTED BY <partition_fields>` to achieve the same purpose as `table.exec.hive.dynamic-grouping.enabled=true`.